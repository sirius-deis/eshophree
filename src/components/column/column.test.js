import { render } from '@testing-library/react';
import Column from './column';

describe('Column Component', () => {
  it("should match snapshot", () => {
    const { container } = render(<Column />);
    expect(container).toMatchSnapshot();
  })
  it('should render children correctly', () => {
    const { getByText } = render(
      <Column>
        <div>Test Child</div>
      </Column>
    );

    expect(getByText('Test Child')).toBeInTheDocument();
  });

  it('should apply the correct styles', () => {
    const { container } = render(
      <Column>
        <div>Styled Child</div>
      </Column>
    );

    const columnElement = container.firstChild;
    expect(columnElement).toHaveStyle('display: flex');
    expect(columnElement).toHaveStyle('flex-direction: column');
  });
});