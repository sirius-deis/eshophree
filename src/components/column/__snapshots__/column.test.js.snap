// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Column Component should match snapshot 1`] = `
<div>
  <div
    class="sc-bgqQcB lgCwwW"
  />
</div>
`;
